app/code/Magento/QuoteGraphQl/Model/Cart/AddSimpleProductToCart.php

```diff
@@ -156,4 +158,17 @@ private function createBuyRequest(float $qty, array $customOptions): DataObject
             ],
         ]);
     }
+
+    /**
+     * @param string $value
+     * @return string|array
+     */
+    private function convertCustomOptions(string $value)
+    {
+        if (substr($value, 0, 1) === "[" ||
```

Comment:
Do we need `OR` or `AND` condition?

Reply:
My mistake. We need AND condition.  PR was updated